content/blog/2025/kai-release-announcement/index.md

```diff
@@ -0,0 +1,62 @@
+---
+author: SavithaRaghunathan, JonathanRecinos
+date: 2025-04-01
+description: >-
+    After months of ideas, development, and feedback from early users and contributors, we're excited to share the first official release of Konveyor AI 0.1.0
+featured: true
+image: header.png
+tags:
+    - Konveyor
+    - Kai
+    - GenAI
+    - AppModernization
+title: "Konveyor AI 0.1.0 is here!"
+---
+
+**Author**: [Savitha Raghunathan](https://github.com/savitharaghunathan), [Jonathan Recinos](https://github.com/JonathanR19)
+
+After months of ideas, development, and feedback from early users and contributors, we're excited to share the first official release of **[Konveyor AI 0.1.0](https://github.com/konveyor/editor-extensions/releases/tag/v0.1.0)**.
```

Comment:
Add something like:

Please download it and try it out in a test environment. Report your issues and feature requests, and contribute fixes and documentation if you can!

Links to places to report/contribute would be good.

Reply:
there is a section at the end with a sub-heading "Try it out!". It has all these details mentioned.

Reply:
that's after a lot of scrolling.  It's helpful to have a call to action near the beginning as well.